catalog/app/containers/Bucket/PackageDialog/Skeleton.js

```diff
@@ -13,19 +13,19 @@ export function FilesInputSkeleton({ animate, className }) {
   return (
     <div className={className}>
       <Skeleton {...{ height: 24, width: 64, animate }} />
-      <Skeleton {...{ height: 320, mt: 2, animate }} />
+      <Skeleton {...{ height: 336, mt: 2, animate }} />
     </div>
   )
 }
 
-export function MetaInputSkeleton({ animate, className }) {
-  return (
-    <div className={className}>
-      <M.Box display="flex" mb={2}>
-        <Skeleton {...{ height: 24, width: 64, animate }} />
-        <M.Box flexGrow={1} />
-        <Skeleton {...{ height: 24, width: 64, animate }} />
-      </M.Box>
+export const MetaInputSkeleton = React.forwardRef(({ animate, className }, ref) => (
+  <div className={className}>
+    <M.Box display="flex" mb={2}>
+      <Skeleton {...{ height: 24, width: 64, animate }} />
+      <M.Box flexGrow={1} />
+      <Skeleton {...{ height: 24, width: 64, animate }} />
+    </M.Box>
+    <div ref={ref}>
```

Comment:
why is ref forwarded to some inner element instead of the root?

Reply:
also, why do we need this div wrapper around the Box?

Reply:
> why is ref forwarded to some inner element instead of the root?

I'm interested in height of this inner element. This inner element's height is a matter of change. It contains editor rows itself, so I think it's semantically correct. 
We can change `ref` to `editorRef` or something like this.

> why do we need this div wrapper around the Box?

There two Boxes:
```jsx
<div ref={wantToKnowHeightHere}><M.Box /><M.Box/></div>
```